Martium.GuessTheRiddleGame/Program.cs

```diff
@@ -83,6 +92,13 @@ static int StartGame()
             return progress;
         }
 
+        private static void ShuffleDictionary()
```

Comment:
Method names is better to reflect purpose, but not reflect implementation details. so better name for this method would be `ShuffleRiddles` (because you intention is randomize riddles, and you achieve this by shuffling dictionary, but its implementation details)